client/src/components/Kanban/Column/Column.tsx

```diff
@@ -9,7 +9,7 @@ type ColumnProps = IColumn;
 const Column = ({ id, name, cards }: ColumnProps): JSX.Element => {
   const classes = useStyles();
   return (
-    <Grid item>
+    <Grid xs={12} md={6} lg="auto" item className={classes.columnGridItem}>
```

Comment:
Looks good! Just curious thoughm why `lg="auto"` here vs `lg={12}`?

Reply:
If I did `lg={12}` a column would span fully across the screen. I'm going to see what "auto" is for medium however, I do think this is a good start.